weeks/week07/week07.md

```diff
@@ -1,3 +1,38 @@
 # Week 7: Web Automation with Playwright
 
-Coming soon.
+## General information:
+Playwright 
+It is the powerful tool for Web Test automation (UI and API) in different programming languages:
+-TypeScript, 
+- JavaScript,
+- .NET,
+- Java.
+- Python
+
+It is 
+- Cross-browser,
+- Cross-platform,
+- Support mobile testing
+
+The scope of this page: Playwright - Python
+
+## Playwright - Python General information:
+Convenient for Web and API testing
```

Comment:
As you list the capabilities, use the list items for better formatting.

Reply:
@ericrommel Thanks. It is done. Check it please.